packages/snaps-execution-environments/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.30.0]
+### Changed
+- No changes this release.
```

Comment:
I think we should mention that SES is now inlined in the index.html: https://github.com/MetaMask/snaps-monorepo/commit/3acbd48b7fdc67717659e8b32e4de4f6bd79e41b